packages/client/src/utils/index.ts

```diff
@@ -95,6 +95,20 @@ export const isValidPostalcode = (value?: string) => {
   return !!(value && value.toString().trim().match(postalCodeRegex));
 };
 
+/**
+ *
+ * This function transforms some cases where a user for example makes a typo or makes a strange space combination to a valid option.<
+ *
+ * @param {string} value
+ */
+export const transformHousenumberToValid = (value: string) =>
+  !/[a-z]/i.test(value) // First we test if your string contains a letter. If not, return as is, there is no need processing it
+    ? value
+    : (value?.toUpperCase().match(/[^a-z\s]+|[a-z]/gi) || []) // tokenize the string into letter/non-letter & non-whitespace chunks
```

Comment:
Would be cleaner if you remove the `!` in the beginning of you ternary and swap the two return values.

Reply:
I don't think so. 
That would turn out to this.

```
export const transformHouseNumberToValid = (value: string) =>
  /[a-z]/i.test(value) // First we test if your string contains a letter. If not, return as is, there is no need processing it
    ? (value?.toUpperCase().match(/[^a-z\s]+|[a-z]/gi) || []) // tokenize the string into letter/non-letter & non-whitespace chunks
        .join("") // Revert to one string
        .replace(/(?<=\D)|(?=\D)/g, " ") // Add spaces before/after a non-digit
        .trim(); // Trim space before and after
    : value

```